state/remote/swift.go

```diff
@@ -131,6 +136,35 @@ func (c *SwiftClient) validateConfig(conf map[string]string) (err error) {
 	}
 	c.path = path
 
+	if archivepath, ok := conf["archive_path"]; ok {
+		log.Printf("[DEBUG] Archivepath set, enabling object versioning")
+		c.archive = true
+		c.archivepath = archivepath
+	}
+
+	if expire, ok := conf["expire_after"]; ok {
+		log.Printf("[DEBUG] Requested that rmemote state expires after %s", expire)
```

Comment:
I was about to merge this but saw the typo and now can't unsee it. Can you do a quick update and I'll merge right after?